extensions/wikia/AdEngine/js/AdTracker.js

```diff
@@ -9,12 +14,12 @@ define('ext.wikia.adEngine.adTracker', ['ext.wikia.adEngine.utils.timeBuckets','
 	function encodeAsQueryString(extraParams) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)